web/src/left_sidebar_navigation_area_popovers.js

```diff
@@ -154,6 +155,10 @@ export function initialize() {
                     }),
                 ),
             );
+            $popper.one("click", "#mark_all_messages_as_read", () => {
+                unread_ops.confirm_mark_all_as_read();
+                instance.hide();
+            });
```

Comment:
I think we've been doing all of our popover click handlers in `onMount` hooks, not `onShow`; probably a good idea to do that here for consistency. And I wouldn't object to a shared `register_mark_all_read_handler(instance)` function that all 3 of these call, to encapsulate that this option is meant to be the same component available in all 3 popovers.